### ISSUE TITLE
Quick Stage and Commit fail with crash due to missing object attribute

### ISSUE THREAD

**Issue by deathaxe** [divmain/GitSavvy#1000]
`git: quick stage` and `git: quick commit` fail with following error:

Console output

```
Traceback (most recent call last):
  File "C:\Apps\Sublime Text 3\Data\Installed Packages\GitSavvy.sublime-package\core/commands/quick_commit.py", line 23, in <lambda>
  File "C:\Apps\Sublime Text 3\Data\Installed Packages\GitSavvy.sublime-package\core/commands/quick_commit.py", line 27, in on_done
AttributeError: 'GsQuickCommitCommand' object has no attribute 'view'
```

- ST3176 x64
- Win10x64 / Win7x64
- git for Windows 2.18
- GitSavvy 1.17.5

The debug log says just nothing here. Enabled it before calling `git: quick stage`, but seems to read status only.

```
[
  {
    "type": "git", 
    "seconds": 0.08849501609802246, 
    "stdin": null, 
    "stderr": "", 
    "command": [
      "status", 
      "--porcelain", 
      "-z"
    ], 
    "stdout": "UU Report 3.txt\u0000A  Report 5.txt\u0000 M submod1\u0000"
  }, 
  {
    "type": "git", 
    "seconds": 0.08434009552001953, 
    "stdin": null, 
    "stderr": "", 
    "command": [
      "status", 
      "--porcelain", 
      "-z"
    ], 
    "stdout": "UU Report 3.txt\u0000A  Report 5.txt\u0000 M submod1\u0000"
  }
]
```

**Comment by stoivo** [comment]
Thanks for reporting, I think it is a duplicate of https://github.com/divmain/GitSavvy/issues/996. Can you try to update to 2.17.5?

**Comment by randy3k** [comment]
Its been reported and should have been fixed with v2.17.5. Report back if it’s not. Thanks.

**Comment by randy3k** [comment]
Oops, the bug was not completely fixed https://github.com/divmain/GitSavvy/blob/master/core/commands/quick_commit.py#L49

**Comment by randy3k** [comment]
@deathaxe It should be fixed by v2.17.6.
:)
